frontend/src/metabase/query_builder/question.ts

```diff
@@ -0,0 +1,32 @@
+import * as Lib from "metabase-lib";
+import type Question from "metabase-lib/v1/Question";
+import { isAdHocModelQuestion } from "metabase-lib/v1/metadata/utils/models";
+
+export function isQuestionDirty(
```

Comment:
Types are incorrect - they should include `undefined` for both `question` and `originalQuestion`

Reply:
updated, and added a `!originalQuestion` check to the predicate in case that is undefined.